perception/autoware_elevation_map_loader/src/elevation_map_loader_node.cpp

```diff
@@ -62,26 +62,26 @@ namespace autoware::elevation_map_loader
 ElevationMapLoaderNode::ElevationMapLoaderNode(const rclcpp::NodeOptions & options)
 : Node("elevation_map_loader", options)
 {
-  layer_name_ = this->declare_parameter("map_layer_name", std::string("elevation"));
-  param_file_path_ = this->declare_parameter("param_file_path", "path_default");
-  map_frame_ = this->declare_parameter("map_frame", "map");
-  bool use_sequential_load = this->declare_parameter<bool>("use_sequential_load", true);
-  int sequential_map_load_num_int = this->declare_parameter<int>("sequential_map_load_num", 1);
+  layer_name_ = this->declare_parameter<std::string>("map_layer_name");
+  param_file_path_ = this->declare_parameter<std::string>("param_file_path");
+  map_frame_ = this->declare_parameter<std::string>("map_frame");
+  bool use_sequential_load = this->declare_parameter<bool>("use_sequential_load");
+  int sequential_map_load_num_int = this->declare_parameter<int>("sequential_map_load_num");
   if (sequential_map_load_num_int > 0) {
     sequential_map_load_num_ = (unsigned int)sequential_map_load_num_int;
   } else {
     throw std::runtime_error("sequential_map_load_num should be larger than 0.");
   }
-  use_inpaint_ = this->declare_parameter("use_inpaint", true);
-  inpaint_radius_ = this->declare_parameter("inpaint_radius", 0.3);
+  use_inpaint_ = this->declare_parameter<bool>("use_inpaint");
+  inpaint_radius_ = this->declare_parameter<double>("inpaint_radius");
```

Comment:
@oguzkaganozt 

Why is `inpaint_radius_` double?
cf. https://github.com/autowarefoundation/autoware.universe/blob/7382d666d6dda9c629184e787e4583204781f413/perception/autoware_elevation_map_loader/src/elevation_map_loader_node.hpp#L106